### DISCUSSION TITLE
What a great project!

### DISCUSSION THREAD

**Discussion by mechpanos** [raomin/ESPAltherma#225] [General]
Hi! I am a Daikin Altherma 3 Monoblock Heat Pump owner, and Homeassistant enthusiast! I have integrated my Daikin heat pump in HA with the HACS integration, I have the Wi-Fi adapter. The integration communicates with the Heat Pump using the closed API of Daikin (same as official Onecta app). So I can control my heat pump, but the information I can get is the bare minimal (set point, outside temperature, operating status more or less). I want to have full information, as I have in the controller! I want to know for example, the pump speed (l/h), the leving and returning water temperature, determine the instant COP etc... So I came across this project!!! What a nice surprise, what a great project this is! 
Congratulations! 
I think you will see my project, soon!

**Comment by raomin** [comment]
Thanks Panos for the positive feedback.
Enjoy ESPAltherma and kali xronia!
